src/app/shared/filters-buttons/purchase-offers-adidas/purchase-offers-adidas.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component} from '@angular/core';
+import { Router } from '@angular/router';
+import { PurchaseOfferService } from '../../../modules/offers/purchase-offers/services/purchase-offer.service';
+
+@Component({
+  selector: 'app-purchase-offers-adidas',
+  templateUrl: './purchase-offers-adidas.component.html',
+  styleUrls: ['./purchase-offers-adidas.component.scss']
+})
+export class PurchaseOffersAdidasComponent {
```

Comment:
Tutaj też 5 komponentów które robią to samo, powinien być jeden który przyjmuje jako parametr identyfikator po którym chcemy filtrować